charts/hono/README.md

```diff
@@ -116,6 +116,13 @@ The command removes all the Kubernetes components associated with the chart and
   *extraVolumeMounts* properties which allow mounting any kind of volume supported by kubernetes into the container file
   system.
 
+### 2.0.4
+
+* *livenessProbeInitialDelaySeconds* and *readinessProbeInitialDelaySeconds* have been deprecated due to the introduction of the
+  *probes* parameter which allows you to configure the HTTP GET probes more specifically instead of just the initialDelaySeconds. The
+  *probes* parameter is configurable globally but can be overwritten per component.
+
```

Comment:
the text is great but can you please move it *before* the 2.0.0 release so that the most recent news are always at the top?

Reply:
updated